dnf-docker-test/README.md

```diff
@@ -12,25 +12,30 @@ correctly. In overall the system is written in such a way that it is package-man
 agnostic, so plugging in tests for `PackageKit` is just about providing correct
 CLI arguments.
 
-## Usage
+## Execute
 
-Install:
-```
-$ git clone https://github.com/j-mracek/dnf_docker_test.git
-$ cd dnf_docker_test/
-$ sudo dnf install docker python3-rpmfluff
-$ sudo docker build -t jmracek/dnftest:1.0.1 .
-```
+Tests can be executed using -dnf_docker_test option for ci-dnf-stack
 
-Execute test from directory with ci-dnf-stack.log:
+Example:
 ```
-$ sudo ./some_path/test-launcher.py test-1
+python2 ../ci-dnf-stack/cidnfstack.py -dnf_docker_test build dnf-pull-requests tito)
```

Comment:
the convention in UNIX is to have double dash for long switch name and one dash for shortcut (usually one character long)